Tools/src/create_rpm.sh

```diff
@@ -47,6 +47,12 @@ chmod ug+rx ${BUILD_ROOT}/SOURCES/opt/aws/amazon-cloudwatch-agent/bin/amazon-clo
 chmod ug+rx ${BUILD_ROOT}/SOURCES/opt/aws/amazon-cloudwatch-agent/bin/start-amazon-cloudwatch-agent
 tar -zcvf ${BUILD_ROOT}/SOURCES/amazon-cloudwatch-agent.tar.gz -C ${BUILD_ROOT}/SOURCES opt etc
 
+echo "*************************************************"
+echo "Saving the tar file for Amazon Linux and RHEL ${ARCH}"
+echo "*************************************************"
```

Comment:
I think for consistency, we should remove the extra echos here. The 
```bash
echo "*************************************************"
```
usage is only at the beginning of the script.

Reply:
Files in the extracted tar file: [ec2-user@ip-172-31-1-251 del]$ tar -xvf ../amazon-cloudwatch-agent.tar.gz 
opt/
opt/aws/
opt/aws/amazon-cloudwatch-agent/
opt/aws/amazon-cloudwatch-agent/logs/
opt/aws/amazon-cloudwatch-agent/var/
opt/aws/amazon-cloudwatch-agent/etc/
opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.d/
opt/aws/amazon-cloudwatch-agent/etc/common-config.toml
opt/aws/amazon-cloudwatch-agent/bin/
opt/aws/amazon-cloudwatch-agent/bin/CWAGENT_VERSION
opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent
opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl
opt/aws/amazon-cloudwatch-agent/bin/config-translator
opt/aws/amazon-cloudwatch-agent/bin/config-downloader
opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-config-wizard
opt/aws/amazon-cloudwatch-agent/bin/start-amazon-cloudwatch-agent
opt/aws/amazon-cloudwatch-agent/doc/
opt/aws/amazon-cloudwatch-agent/doc/amazon-cloudwatch-agent-schema.json
opt/aws/amazon-cloudwatch-agent/LICENSE
opt/aws/amazon-cloudwatch-agent/NOTICE
opt/aws/amazon-cloudwatch-agent/THIRD-PARTY-LICENSES
opt/aws/amazon-cloudwatch-agent/RELEASE_NOTES
etc/
etc/init/
etc/init/amazon-cloudwatch-agent.conf
etc/systemd/
etc/systemd/system/
etc/systemd/system/amazon-cloudwatch-agent.service